[15:26:44]: ContextError: Service provided incorrect authorization certificate
(Verifier: did:key:zTL5kCGeFa3dV3BFNxfwzjyMWYquThnNubCgzsnrGCiBFJWJLnLZayYAsoYXmazobCDPubwbU2m763HAsJuvHnYuM6UueoN4jXpVn5RCaMTb7rWxtcxKsyLFJPskuAXpfAPX9h3EgDQwcKeoxTPbcfD25BickKrwt5rvv6SF4urhbzLNLxwthAGc2Jdq9hY7zxMa8AdqHmRYTW6KYu4G31oUrQyyDYYG8d2kw2Gku3H3u85ik5zyxi6Kwiki5XUeGNYkvv2PJgWpBKXN29yLugVk95MQBWqp7PHDnidKWmDq7RTYnUtQGkN7Qw2fyU1BzexKgx1ohhhJ2xTmQoU1xE8LeaAn1dNkF5nbEtumy67aq43AzScQ)

